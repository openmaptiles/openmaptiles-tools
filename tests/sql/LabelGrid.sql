-- LabelGrid
SELECT LabelGrid(ST_GeomFromText('POINT(100 -100)',900913), 64*9.5546285343) AS v1;	-- POINT(305.7481130976 -305.7481130976)
