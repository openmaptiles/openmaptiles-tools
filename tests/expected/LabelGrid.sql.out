                  v1                   
---------------------------------------
 POINT(305.7481130976 -305.7481130976)
(1 row)

